sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -899,24 +899,15 @@ object JdbcUtils extends Logging with SQLConfHelper {
       schema: StructType,
       caseSensitive: Boolean,
       options: JdbcOptionsInWrite): Unit = {
+    val statement = conn.createStatement
     val dialect = JdbcDialects.get(options.url)
     val strSchema = schemaString(
       schema, caseSensitive, options.url, options.createTableColumnTypes)
-    val createTableOptions = options.createTableOptions
-    // Create the table if the table does not exist.
-    // To allow certain options to append when create a new table, which can be
-    // table_options or partition_options.
-    // E.g., "CREATE TABLE t (name string) ENGINE=InnoDB DEFAULT CHARSET=utf8"
-    val sql = s"CREATE TABLE $tableName ($strSchema) $createTableOptions"
-    executeStatement(conn, options, sql)
-    if (options.tableComment.nonEmpty) {
-      try {
-        executeStatement(
```

Comment:
We can still leave the table comment statement execution here, unless there is a use case to allow dialect to override it as well.